app/src/main/java/net/gsantner/markor/activity/MainActivity.java

```diff
@@ -411,8 +411,7 @@ public GsFileBrowserOptions.Options getFilesystemFragmentOptions(GsFileBrowserOp
                 public void onFsViewerConfig(GsFileBrowserOptions.Options dopt) {
                     dopt.descModtimeInsteadOfParent = true;
                     dopt.rootFolder = _appSettings.getNotebookDirectory();
-                    final File fallback = _appSettings.getFolderToLoadByMenuId(_appSettings.getAppStartupFolderMenuId());
-                    final File file = MarkorContextUtils.getValidIntentFile(getIntent(), fallback);
+                    final File file = MarkorContextUtils.getValidIntentFile(getIntent(), dopt.rootFolder);
```

Comment:
Does Markor still startup with i.e. Favorites or storage if you select that in the filebrowser app settings?